src/ConstructiveSolidGeometry/VolumePrimitives/Cone.jl

```diff
@@ -99,37 +99,44 @@ get_φ_limits(c::Cone{T, <:Any, <:AbstractInterval, <:Any}) where {T} = (c.φ.le
 
 get_z_limits(c::Cone{T}) where {T} = (_left_linear_interval(c.z), _right_linear_interval(c.z))
 
-function _get_decomposed_surfaces_cone(c::Cone{T}, rbotMin, rbotMax, rtopMin, rtopMax, zMin, zMax) where {T}
+function _is_cone_collapsed(rbotMin::T, rbotMax::T, rtopMin::T, rtopMax::T, zMin::T, zMax::T) where {T}
```

Comment:
Can well call this something like `_is_2D_cone`? I find `collapsed` a strange wording for this.. Or is that 
the scientific term I don't know about?

Can you briefly state the purpose of this function, just to be sure that I understand it right?

Reply:
Its to check if one of the dimensions of the cone is zero. For example if the cone volume collapsed into a cone mantle or cylindrical annulus. The term collapsed is commonly used for these 3D to 2D transformations